m3u8/model.py

```diff
@@ -445,6 +442,15 @@ def dumps(self, last_segment):
                 output.append(str(self.key))
                 output.append('\n')
 
+        if last_segment and self.init_section != last_segment.init_section:
+            assert self.init_section, "init section can't be None if previous is not None"
```

Comment:
Not sure if this is the right way to propagate an error.
Users of this feature may need to catch it and handle the lack of a value in `init_section`.
What is your opinion?

Reply:
Yep! You are right.
This condition should be always true for parsed playlists, but may be violated for user crafted playlist.